packages/integration-tests/cmr.js

```diff
@@ -96,7 +97,7 @@ async function conceptExists(cmrLink) {
  * @returns {Promise<undefined>}
  * @throws {TimeoutError} - throws error when timeout is reached
  */
-async function waitForConceptExistsOutcome(cmrLink, expectation, retries = 3, interval = 2000) {
+async function waitForConceptExistsOutcome(cmrLink, expectation, retries = 20, interval = 3000) {
```

Comment:
I'm not sure this function needs to be as configurable as it is.  In fact, I think we should remove the retries and interval options.

You've put a lot of work into figuring out what the correct time to wait for a valid response, and how often to retry.  I think we should embed that knowledge here in this function instead of spreading it throughout the code.

This function is only called in two places, so that will be an easy change to make.  (Incidentally, those two places are both using different intervals than each other, and different than the default that you're setting here.)